pkg/storage/storage.go

```diff
@@ -19,24 +20,21 @@ import (
 	"time"
 
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	apitypes "k8s.io/apimachinery/pkg/types"
 	"k8s.io/klog/v2"
 	"k8s.io/metrics/pkg/apis/metrics"
 
 	"sigs.k8s.io/metrics-server/pkg/api"
 )
 
-// kubernetesCadvisorWindow is the max window used by cAdvisor for calculating
-// CPU usage rate.  While it can vary, it's no more than this number, but may be
-// as low as half this number (when working with no backoff).  It would be really
-// nice if the kubelet told us this in the summary API...
-var kubernetesCadvisorWindow = 30 * time.Second
-
 // storage is a thread save storage for node and pod metrics
 type storage struct {
-	mu    sync.RWMutex
-	nodes map[string]NodeMetricsPoint
-	pods  map[apitypes.NamespacedName]PodMetricsPoint
+	mu        sync.RWMutex
+	nodes     map[string]NodeMetricsPoint
```

Comment:
I noticed that your store implementation only stores points if they are newer then already stored and cpuUsageOverTime implementation assumes (doesn't check if window != 0). Let's document this invariant.

Reply:
I added a note